common/src/main/java/whocraft/tardis_refined/common/blockentity/door/BulkHeadDoorExtensionBlockEntity.java

```diff
@@ -0,0 +1,87 @@
+package whocraft.tardis_refined.common.blockentity.door;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.NbtUtils;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+import whocraft.tardis_refined.common.block.door.BulkHeadDoorBlock;
+import whocraft.tardis_refined.registry.TRBlockEntityRegistry;
+
+public class BulkHeadDoorExtensionBlockEntity extends BlockEntity {
+
+    private BlockPos masterDoorBlockPos;
+    private BulkHeadDoorBlockEntity masterDoorBlockEntity;
+
+    public BulkHeadDoorExtensionBlockEntity(BlockPos blockPos, BlockState blockState) {
+        super(TRBlockEntityRegistry.BULK_HEAD_DOOR_EXT.get(), blockPos, blockState);
+    }
+
+    @Override
+    protected void saveAdditional(CompoundTag compoundTag) {
+        super.saveAdditional(compoundTag);
+
+        if (masterDoorBlockPos != null) {
+            compoundTag.put("MASTER_DOOR", NbtUtils.writeBlockPos(this.masterDoorBlockPos));
+        }
+
+    }
+
+    @Override
+    public void load(CompoundTag compoundTag) {
+        CompoundTag entityLocation = compoundTag.getCompound("MASTER_DOOR");
```

Comment:
Usage of tag should be made in the constants class we have, should also be lowercased

Reply:
Made a tag